### PR TITLE
Feature/back 6 anynote django application

### PR DESCRIPTION
**Pull request by egorgur**
Anynote main django application layout

```sh
src
    ├───apps
    │   ├───anynote
    │        ├───api
    │        ├───migrations
    │   ├───authentication
    │        ├───api
    │        ├───migrations
    ├───config
    │   ├───settings
    │        ├───templates
    ├───general
    │   ├───tests
    │        ├───fixtures
    │   ├───utils
    ├───tests
```

### CODE REVIEW

File: docker-compose.yaml
Comment (egorgur) [inline docker-compose.yaml:1]: Ignore this, there are actual changes

File: src/config/settings/logging.py
Comment (IWorkForFood) [inline src/config/settings/logging.py:1]: Is not it less readable

```
"handlers": {
        "console": {"level": "INFO", "class": "rich.logging.RichHandler", "formatter": "standard", "filters": []},
```
than this one

```
"handlers": {
        "console": {"level": "INFO",
         "class": "rich.logging.RichHandler", 
         "formatter": "standard", "filters": []
         },
```

The same question applies to this part:

```
"formatters": {"standard": {"format": "%(asctime)s %(levelname)s %(name)s %(message)s"}},
```
and to this part:

```
"filters": {"info_filter": {"()": "src.config.settings.InfoFilter"}},
```

Comment (egorgur) [reply src/config/settings/logging.py:1]: I agree with you and i tried to make it unfold manually. But the ruff pre-commit formatter makes it fold back. Also just standart ctrl + alt + l also makes them shrink.
So i could not unfold those dictionaries without automatically folding them back by the linter

Comment (egorgur) [reply src/config/settings/logging.py:1]: resolved in IRL

**Pull request merged**